.github/workflows/container-scan.yml

```diff
@@ -309,3 +309,107 @@ jobs:
         uses: github/codeql-action/upload-sarif@v2
         with:
           sarif_file: snyk-code.sarif
+
+  scan-syft-requirements:
+    permissions:
+      contents: read # for actions/checkout to fetch code
+      security-events: write # for github/codeql-action/upload-sarif to upload SARIF results
+      actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      #Generate SBOM
+      - name: Generate SBOM
+        run: |
+          pip install cyclonedx-bom
+          pip freeze > requirements.txt
```

Comment:
Do we need to pip install syft from the source folder first?

Reply:
fixed this